libglusterfs/src/inode.c

```diff
@@ -732,7 +741,8 @@ inode_new(inode_table_t *table)
             table->lru_size++;
             GF_ASSERT(!inode->in_lru_list);
             inode->in_lru_list = _gf_true;
-            __inode_ref(inode, false);
+            if (!root_with_ref)
```

Comment:
Isn't this always true ?

Reply:
Missed it while removing the check for root's ref in the earlier commit. Fixed now.